app/views/admin_outgoing_message/edit.html.erb

```diff
@@ -15,6 +15,14 @@
           <%= text_field 'outgoing_message', 'tag_string', class: 'span6' %>
         </div>
       </div>
+
+      <div class="control-group">
+        <label class="control-label" for="outgoing_message_from_name">From name</label>
+
+        <div class="controls">
+          <%= text_field 'outgoing_message', 'from_name', class: 'span6' %>
+        </div>
+      </div>
```

Comment:
On feeling it out, I don't think this is ideal, but will do for now.

One issue is that it's possible to leave a request in an "unnamed" state:

![Screenshot 2023-05-10 at 17 08 48](https://github.com/mysociety/alaveteli/assets/282788/1e864730-cdbc-4b50-9526-e16e0abebb23)

We could add a validation here to ensure it's not blank – that would be an easy fix.

Another is that I don't think we'll want to _edit_ the name; we just want to be able to "erase" it (but we want to do so transparently, and so that the UI makes sense). In the event we did want to "change" it, I think we'd want to retain the original (for archive preservation), but change the published version via a censor rule (so that we have a note, and do our duty to the user).

Better would be more like the new [erase button](https://github.com/mysociety/alaveteli/pull/7727), but scoped to the specific content:

![Screenshot 2023-05-10 at 17 05 29](https://github.com/mysociety/alaveteli/assets/282788/5feef7ee-56a2-4d15-87e0-ad0b0a758c77)

Could take that a step further and make it easier to add a redaction:

![Screenshot 2023-05-10 at 17 36 46](https://github.com/mysociety/alaveteli/assets/282788/b26f37e2-f747-4e29-a0cb-a79a04e2d481)

This definitely feels a bit much for right now though.

Instead, let's just use some explanatory notes.

First, let's move this field under the "Message" fieldset, as the "From name" was also sent to the authority along with the body. That way the existing notes nearly work already. Here's a suggestion for a tweak to the notes themselves:

> **Note:** _Editing messages is not recommended_. Most information can be hidden from publication using censor rules. This allows the original information to be retained for archival purposes.
> 
> Messages can be edited to destroy information included by users that we have no good reason to preserve. Messages will already have been sent to the authority, and replies may also include that information and be automatically published.
> 
> You can also edit a message before resending it. Only the edited version will be shown on the public page.